watchfiles/cli.py

```diff
@@ -93,6 +93,20 @@ def cli(*args_: str) -> None:
         choices=['warning', 'info', 'debug'],
         help='Log level, defaults to "info"',
     )
+    parser.add_argument(
+        '--sigint_timeout',
```

Comment:
Sorry, just noticed, these should use a hyphen to match other arguments.

And below.

Reply:
No worries, change pushed.